src/UI/Factory.php

```diff
@@ -896,4 +896,67 @@ public function symbol(): Component\Symbol\Factory;
      * @return \ILIAS\UI\Component\Toast\Factory
      */
     public function toast(): C\Toast\Factory;
+
+    /**
+     * ---
+     * description:
+     *   purpose: >
+     *     The Launcher Component is used to launch an object, process or workflow. From a
+     *     users' perspective the main goal of the Launcher component is to present just
+     *     relevant information for a quick decision to launch a certain object, process or
+     *     workflow and if all preconditions for launching are fullfilled. For a clear
+     *     guidance of the users' intent it  can present a title and a descriptive text and
```

Comment:
The Launcher starts an object, a process or a workflow. It conveys the smallest set of information needed to decide upon launching. The Launcher communicates clearly whether or not a user can or cannot launch at any given time.